stdpopsim/catalog/GorGor/species.py

```diff
@@ -0,0 +1,166 @@
+import stdpopsim
+from . import genome_data
+
+_recombination_rate = dict()
+_mutation_rate = dict()
+_ploidy = dict()
+
+# genome-wide recombination rate
+# associated with all recombining chromosomes
+_Ne = 25200
+_mean_recombination_rate = 1.193e-08
+_recombination_rate_data = {
+    "1": _mean_recombination_rate,
+    "2A": _mean_recombination_rate,
+    "2B": _mean_recombination_rate,
+    "3": _mean_recombination_rate,
+    "4": _mean_recombination_rate,
+    "5": _mean_recombination_rate,
+    "6": _mean_recombination_rate,
+    "7": _mean_recombination_rate,
+    "8": _mean_recombination_rate,
+    "9": _mean_recombination_rate,
+    "10": _mean_recombination_rate,
+    "11": _mean_recombination_rate,
+    "12": _mean_recombination_rate,
+    "13": _mean_recombination_rate,
+    "14": _mean_recombination_rate,
+    "15": _mean_recombination_rate,
+    "16": _mean_recombination_rate,
+    "17": _mean_recombination_rate,
+    "18": _mean_recombination_rate,
+    "19": _mean_recombination_rate,
+    "20": _mean_recombination_rate,
+    "21": _mean_recombination_rate,
+    "22": _mean_recombination_rate,
+    "X": _mean_recombination_rate,
+    "MT": 0,
+}
+
+# genome-wide average mutation rate
+# associated with all chromosomes
+_mean_mutation_rate = 1.235e-8
+_mutation_rate = {
+    "1": _mean_mutation_rate,
+    "2A": _mean_mutation_rate,
+    "2B": _mean_mutation_rate,
+    "3": _mean_mutation_rate,
+    "4": _mean_mutation_rate,
+    "5": _mean_mutation_rate,
+    "6": _mean_mutation_rate,
+    "7": _mean_mutation_rate,
+    "8": _mean_mutation_rate,
+    "9": _mean_mutation_rate,
+    "10": _mean_mutation_rate,
+    "11": _mean_mutation_rate,
+    "12": _mean_mutation_rate,
+    "13": _mean_mutation_rate,
+    "14": _mean_mutation_rate,
+    "15": _mean_mutation_rate,
+    "16": _mean_mutation_rate,
+    "17": _mean_mutation_rate,
+    "18": _mean_mutation_rate,
+    "19": _mean_mutation_rate,
+    "20": _mean_mutation_rate,
+    "21": _mean_mutation_rate,
+    "22": _mean_mutation_rate,
+    "X": _mean_mutation_rate,
+    "MT": 0,
```

Comment:
This surely should not be 0! I realize the mean rate is not right either, but still.
```suggestion
    "MT": _mean_mutation_rate,
```

Reply:
I've done this change.

Reply:
I agree it shouldn't be zero. The fix is ok.